frontend/src/components/feature/chat/ChatMessage/ActionModals/ReactionAnalyticsModal.tsx

```diff
@@ -0,0 +1,111 @@
+import { useMemo } from "react"
+import { IconButton, Dialog, Flex, Text, Tabs, Box } from "@radix-ui/themes"
+import { BiX } from "react-icons/bi"
+import { ReactionObject } from "../MessageReactions"
+import { ReactionAnalyticsDialogProps } from "../MessageActions/MessageReactionAnalytics"
+import { MdOutlineEmojiEmotions } from "react-icons/md";
+import { useGetUserRecords } from "@/hooks/useGetUserRecords"
+import { UserAvatar } from "@/components/common/UserAvatar"
+import { getUserImage } from "@/utils/operations"
+
+export const ReactionAnalyticsModal = ({ reactions }: ReactionAnalyticsDialogProps) => {
+
+    const { reaction_emojis, all_reacted_members } = useMemo(() => {
+        const reaction_emojis = reactions.map((reaction: ReactionObject) => reaction.reaction);
+        const all_reacted_members = reactions.flatMap(({ reaction, users }: ReactionObject) =>
+            users.map((user: string) => ({ user, reaction }))
+        );
+        return { reaction_emojis, all_reacted_members };
+    }, [reactions]);
+
+    return (
+        <>
+            {/* <DialogHeader /> */}
+            <Tabs.Root defaultValue="All">
+                <Flex direction="column" gap="4">
+                    <Tabs.List>
+                        <TabTrigger emojiStr="All" />
+                        {reaction_emojis.map((emojiStr) => {
+                            const reaction = reactions.find((r) => r.reaction === emojiStr);
+                            return <TabTrigger key={emojiStr} emojiStr={emojiStr} count={reaction?.count} />;
+                        })}
+                    </Tabs.List>
+                    <Box>
+                        <Tabs.Content value="All">
+                            <UserList users={all_reacted_members} />
+                        </Tabs.Content>
+                        {reactions.map((reaction) => (
+                            <Tabs.Content key={reaction.reaction} value={reaction.reaction}>
+                                <UserList users={reaction.users.map((user) => ({ user }))} />
+                            </Tabs.Content>
+                        ))}
+                    </Box>
+                </Flex>
+            </Tabs.Root>
+        </>
+    );
+};
+
+// const DialogHeader = () => (
+//     <Flex justify="between">
+//         <Dialog.Title>
+//             <Flex align="center" gap="2">
+//                 <MdOutlineEmojiEmotions />
+//                 <Text>Reactions</Text>
+//             </Flex>
+//         </Dialog.Title>
+//         <Dialog.Close className="invisible sm:visible">
+//             <IconButton size="1" variant="soft" color="gray">
+//                 <BiX size="18" />
+//             </IconButton>
+//         </Dialog.Close>
+//     </Flex>
+// );
+
+const TabTrigger = ({ emojiStr, count }: { emojiStr: string; count?: number }) => (
+    <Tabs.Trigger value={emojiStr}>
```

Comment:
Without overriding the color, on some browsers the emojis are faded:
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/9f1ed191-1ad2-486b-bcc2-620a6ca66f9c">


```suggestion
    <Tabs.Trigger value={emojiStr} className="text-gray-11">
```

After:
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/21cec9b9-5c8a-4e35-94d2-b01ed4ec05b0">